Misc/largestRange.java

```diff
@@ -0,0 +1,46 @@
+import java.util.*;
+public class largestRange
+{
+    // Finds the length of longest occurring consecutive numbers range in an array
+    public static int longestRange(int[] nums)
+    {
+        int longestRange = 0;
+        HashMap<Integer,Boolean> num = new HashMap<>(); // Stores a mapping of a number to whether the current number is part of a particular consecutive range or not
+        for(int x : nums)
```

Comment:
The code may be failing prettier check because of this. According to pritter as far I know the correct code will be 
`for(int x : nums) num.put(x, true);`

Reply:
Seems I was right, this is the actual format in the final final.